### ISSUE TITLE
[Bug] .snakes draw not working

### ISSUE THREAD

**Issue by lemonsaurus** [python-discord/seasonalbot#218]
`.snakes draw` is currently not providing any output in production. Please investigate and fix.

**Comment by MarkKoz** [comment]
For what it's worth, I did successfully test it locally on the 22nd of March when I was working on cleaning up docker images.

**Comment by sco1** [comment]
Can reproduce this locally outside of the container:

```
Traceback (most recent call last):
  File "/mac/pydis-seasonalbot/.venv/lib/python3.7/site-packages/discord/ext/commands/core.py", line 79, in wrapped
    ret = await coro(*args, **kwargs)
  File "/mac/pydis-seasonalbot/bot/seasons/evergreen/snakes/snakes_cog.py", line 628, in draw_command
    file = File(png_bytes, filename='snek.png')
  File "/mac/pydis-seasonalbot/.venv/lib/python3.7/site-packages/discord/file.py", line 68, in __init__
    self.fp = open(fp, 'rb')
ValueError: embedded null byte
```

This was likely introduced by the dependency updates introduced by #201, since this was working locally with the initial breaking changes migrated in #132. If I had to guess I'd say that Pillow is the culprit (#201 bumped this to `v6.0.0` from `v5.4.1`), should be able to clean it up in a little bit.

**Comment by sco1** [comment]
Should now be fixed by #200
